source/use-cases/sensitive-data-encryption.txt

```diff
@@ -300,6 +289,17 @@ information:
 - The BSON Type of each field (only required for deterministically
   encrypted fields)
 
+.. admonition::JSON Schema Validation
```

Comment:
"Automatic Client-Side Field Level Encryption JSON Schema Does Not Support Document Validation"